state_machine/applications/flight/lib/IGRF.py

```diff
@@ -217,3 +219,27 @@ def igrf(date, latitude_degrees, elongitude_degrees, r_norm_km):
         - [x, y, z] the magnetic field in nanotesla in (North, East, Down)
     """
     return _igrf13_5(gh, date, latitude_degrees, elongitude_degrees, r_norm_km, cl, sl, p, q)
+
+def igrf_eci(date, r_eci):
+    """Returns the fifth order approximation from the IGRF-13 model. 
+    Only contains data from 2020, so it should only be accurate from 2020-2025.
+    IGRF-13 Takes in geocentric coordinates, and outputs in NED (North, East, Down).
+    We solve this by applying the following conversions: ECI->ECEF->GEOC=>[IGRF]=>NED->ECEF->ECI
+
+    Args:
+        - date: A unix timestamp.
+        - r_eci: Earth Centered Interital frame position (km)
+    Returns:
+        - [x, y, z] the magnetic field in nanotesla in ECI (Earth Centered Inertial)
+    """
+    ecef_eci = frames.eci_to_ecef(date)
+    eci_ecef = ecef_eci.transpose()
+
+    r_ecef = np.dot(ecef_eci, r_eci)
+    long, lat, _ = frames.convert_ecef_to_geoc(r_ecef)
+
+    b_ned = igrf(date, lat / np.pi * 180, long / np.pi * 180, np.linalg.norm(r_eci))
```

Comment:
use parens for rad2deg conversion

Reply:
Honestly, this isn't ambiguous in the slightest to me. It's also pretty standard not to put unnecessary parens in places like these.

Reply:
Also it's kinda ugly to have it be 
```python
b_ned = igrf(date, (lat / np.pi) * 180, (long / np.pi) * 180, np.linalg.norm(r_eci))
```

Reply:
I changed it anyway*